source/blog/032-embedded-sass-is-live.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Embedded Sass is Live"
+author: Natalie Weizenbaum
+date: 2022-02-01 2:00 PST
+---
+
+After several years of planning and development, I'm excited to finally announce
+the stable release of Embedded Dart Sass along with its first official wrapper,
+the [`sass-embedded`] package available now on npm!
+
+[`sass-embedded`]: https://www.npmjs.com/package/sass-embedded
+
+Embedded Sass is an ongoing effort to make a highly-performant Sass library
+available to as many different languages as possible, starting with Node.js.
+Although Node.js already has access to the pure-JS `sass` package, the nature of
+JavaScript inherently limits how quickly this package can process large Sass
+files especially in asynchronous mode. We expect `sass-embedded` to be a major
+boon to developers for whom compilation speed is a concern.
+
```

Comment:
For the developers who are still on `node-sass`, it might be worth mentioning `node-sass` explicitly to motivate this post even more. We could briefly mention embedded is intended as an ~equally performant yet up-to-spec implementation, and endorse migration more strongly. (I guess that depends on how forward- or backward-looking you want this post to be.)